src/graph/visitor/ExtractFilterExprVisitor.cpp

```diff
@@ -276,6 +276,7 @@ bool ExtractFilterExprVisitor::visitLogicalOr(LogicalExpression *expr) {
         if (canNotPushedIndex != -1 || !canBeSplit) {
           return false;
         }
```

Comment:
Crashed at `DCHECK(ok())` in ExprVisitorImpl.cpp:130.
Better to fix else branch:
```
 else {
      if (!canBePushed_) {
        return false;
      }
      operands[i]->accept(this);
    }
```
Alternatively, we could do some minor refactoring of the visitor, such as adding additional state variables other than "can" and "cannot" pairs to allow the internal logic to continue.

Reply:
ok

Reply:
Will lead to test fail.